libs/agno/agno/agent/agent.py

```diff
@@ -156,6 +156,8 @@ class Agent:
     enable_session_summaries: bool = False
     # If True, the agent adds session summaries to the context
     add_session_summary_to_context: Optional[bool] = None
+    # Number of runs to include when generating session summary (None = all runs)
+    num_summary_runs: Optional[int] = None
```

Comment:
so the flag sounds more like number of summary runs, which is incorrect. it has to convey more like "number of runs to be used for generating summary"...